test/integration/scheduler_perf/config/performance-config.yaml

```diff
@@ -688,49 +688,37 @@
 #      initPods: 20000
 #      measurePods: 5000
 
+# Measure throughput of regular schedulable pods that are interleaved by unschedulable pods injected at 100/s rate.
 - name: Unschedulable
   workloadTemplate:
   - opcode: createNodes
     countParam: $initNodes
-  - opcode: createPods
-    countParam: $initPods
-    podTemplatePath: config/templates/pod-large-cpu.yaml
-    skipWaitToCompletion: true
+  - opcode: churn
+    mode: create
+    templatePaths:
+    - config/templates/pod-large-cpu.yaml
+    intervalMilliseconds: 10
   - opcode: createPods
     countParam: $measurePods
     podTemplatePath: config/templates/pod-default.yaml
     collectMetrics: true
   workloads:
-  - name: 5Nodes/2InitPods
```

Comment:
Tests that small does not make much sense, because churn may not be able to inject unschedulable nodes at all. Do you think there is a value in having them?

Reply:
The purpose of small tests is just for us to make sure that there aren't bugs in the test implementation. They run along with the rest of the integration suite.

See the comment at the top of this file https://github.com/kubernetes/kubernetes/blob/master/test/integration/scheduler_perf/config/performance-config.yaml

Reply:
This is why I preserved integration-test label on 500Nodes/1kPods, but reverted the smaller test now